modules/gapi/samples/privacy_masking_camera.cpp

```diff
@@ -0,0 +1,215 @@
+#include <chrono>
+#include <iostream>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/gapi.hpp>
+#include <opencv2/gapi/core.hpp>
+#include <opencv2/gapi/imgproc.hpp>
+#include <opencv2/gapi/infer.hpp>
+#include <opencv2/gapi/render.hpp>
+#include <opencv2/gapi/infer/ie.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+#include <opencv2/gapi/streaming/cap.hpp>
+#include <opencv2/highgui.hpp>
+
+const std::string about =
+    "This is an OpenCV-based version of Privacy Masking Camera example";
+const std::string keys =
+    "{ h help |   | print this help message }"
+    "{ input  |   | Path to an input video file }"
+    "{ platm  |   | IE vehicle/license plate detection model IR }"
+    "{ platw  |   | IE vehicle/license plate detection model weights }"
```

Comment:
It's convenient for OpenVINO samples to specify just path to `.xml` and keep both `.xml` and `.bin` with same same name in the same directory. So probably we can just estimate `.bin` path from `.xml` and then only `plat` flag is needed.

Reply:
I remember a couple of DNN samples do that already, do we have a function for this somewhere in utils?

Reply:
Added my own